bot/bot.py

```diff
@@ -77,28 +77,39 @@ async def remove_word(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
 
 
-def get_random_quiz():
+def get_random_quiz(language=None):
     filename_json = 'words.json'
 
     # Load the JSON file
     with open(filename_json, 'r', encoding='utf-8') as f:
         word_list = json.load(f)
 
+    # Filter the word list based on the specified language
+    if language:
+        word_list = [word for word in word_list if word.get('language') == language]
+
+    if not word_list:
+        return "No words found for the specified language.", None
+
     # Pick a random word-description pair
     random_pair = random.choice(word_list)
 
     word = random_pair['word']
-    language = random_pair['language']
     description = random_pair['description']
 
-    return f"What is the word (in {language}) with given description: {description}?", word
+    return f"What is the word (in {random_pair['language']}) with given description: {description}?", word
```

Comment:
There is a "language" variable just for that

Reply:
fixed in https://github.com/SWel1a/QuizBot/pull/11/commits/bc84d7b3289c8eee6fd0538a2d871fd03c0a7ec5